src/helpers/accrual-schedule.interface.ts

```diff
@@ -0,0 +1,12 @@
+export interface AccrualSchedule {
+  AccrualCategory: {
+    AccrualCategoryCode: string;
+  };
+  SpreadRate: number;
+  YearBasis: {
+    YearBasisCode: string;
+  };
+  IndexRateChangeFrequency: {
+    IndexRateChangeFrequencyCode: string;
+  };
+}
```

Comment:
Why are `AccrualSchedule`, `BundleAction`, and `RepaymentSchedule` all in `src/helpers`? I expected to see them in `src/modules/acbs`. We can add a folder to `acbs` for bundles if we need

Reply:
Moved to `acbs/dto/bundle-action`.